lib/form_personal_details.dart

```diff
@@ -0,0 +1,200 @@
+                textAlign: TextAlign.left,
+                style: TextStyle(
+                  color: primaryColor,
+                  fontSize: 26,
+                  fontWeight: FontWeight.w600,
+                  height: 1.5,
+                ),
+              ),
+              const Text(
+                'Enter personal information below',
+                style: TextStyle(color: primaryColor, fontSize: 16),
+              ),
+              const SizedBox(height: 32),
+              FormBox(
+                icon: Icons.person_2_outlined,
+                hintText: 'First Name',
+                controller: _firstNameController,
+                keyboardType: TextInputType.name,
+              ),
+              // FIRST NAME ERROR
+              if (_firstNameError.isNotEmpty) ...[
+                const SizedBox(height: 16),
+                Text(
+                  _firstNameError,
+                  style: const TextStyle(color: Colors.red, fontSize: 16),
+                ),
+              ],
+              const SizedBox(height: 16),
+              FormBox(
+                icon: Icons.person_2_outlined,
+                hintText: 'Last Name',
+                controller: _lastNameController,
+                keyboardType: TextInputType.name,
+              ),
+              // LAST NAME ERROR
+              if (_lastNameError.isNotEmpty) ...[
+                const SizedBox(height: 16),
+                Text(
+                  _lastNameError,
+                  style: const TextStyle(color: Colors.red, fontSize: 16),
+                ),
+              ],
+              const SizedBox(height: 16),
+              FormBox(
+                icon: Icons.calendar_month_outlined,
+                hintText: 'Date Of Birth',
+                controller: _dateOfBirthController,
+                keyboardType: TextInputType.number,
+              ),
+              // DATE OF BIRTH ERROR
+              if (_dateOfBirthError.isNotEmpty) ...[
+                const SizedBox(height: 16),
+                Text(
+                  _dateOfBirthError,
+                  style: const TextStyle(color: Colors.red, fontSize: 16),
+                ),
+              ],
+              const SizedBox(height: 16),
+              FormBox(
+                icon: Icons.phone_outlined,
+                hintText: 'Mobile Number',
+                controller: _mobileNumberController,
+                keyboardType: TextInputType.phone,
+                // TODO: Use input formatters to modify keyboard interface?
+                // inputFormatters: <TextInputFormatter>[
+                //   FilteringTextInputFormatter.digitsOnly,
+                //   LengthLimitingTextInputFormatter(12), // Adjust according to your phone number format
+                //   also try MaskTextInputFormatter to enforce a specific format
+                // ],
+              ),
+              // MOBILE NUMBER ERROR
+              if (_mobileNumberError.isNotEmpty) ...[
```

Comment:
We discussed in the call, taking this and making a generic method that takes an error string and outputs the list.